templates/docs/examples/patterns/code-snippet/_script.js

```diff
@@ -0,0 +1,26 @@
+function attachDropdownEvents(dropdown) {
```

Comment:
I think we tend to add a bit of a jsdoc comment on top of those in other examples, to give better idea of what the function does and what the params mean.
Could you have a look at some other example JS and use similar comment style?